jupyterlab/labbox_ephys_widgets_jp/src/pluginWidgets.tsx

```diff
@@ -377,17 +378,18 @@ export class SortingView extends DOMWidgetView {
     }
 
     return (
-      <PluginComponentWrapper
-        plugin={plugin}
-        hither={hither}
-        sortingObject={sortingObject}
-        recordingObject={recordingObject}
-        sortingInfo={sortingInfo}
-        recordingInfo={recordingInfo}
-        plugins={filterPlugins(plugins)}
-        calculationPool={calculationPool}
-        model={this.model}
-      />
+      <HitherContext.Provider value={hither}>
```

Comment:
The hither call mechanism is simplified by using a context provider and a useContext() hook. With this new system, we don't need to pass the hither context in the props from parent->child components.